src/System Application/App/Document Processor/src/PDFDocumentProcessor.Codeunit.al

```diff
@@ -0,0 +1,33 @@
+
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.IO;
+
+using System.Utilities;
+
+/// <summary>
+/// Codeunit that provides helper functions for PDF processing.
+/// </summary>
+codeunit 3110 "PDF Document Processor"
+{
+    Access = Public;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        PDFHelperImpl: Codeunit "PDF Document Processor Impl.";
+
+    /// <summary>
+    /// This procedure is used to convert a PDF file to an image.
+    /// </summary>
+    /// <param name="DocumentStream">Stream of the PDF file.</param>
+    /// <param name="ImageStream">Stream of the image file.</param>
+    /// <param name="ImageFormat">Image format to convert the PDF to.</param>
+    /// <param name="PageNumber">Page number to convert.</param>
+    procedure ConvertToImage(DocumentStream: InStream; var ImageStream: InStream; ImageFormat: Enum "Image Format"; PageNumber: Integer)
```

Comment:
The current implementation does not make much sense.

We should at least return the number of pages that the document has. 
Consider adding a method that converts the first page, so page number is not needed.

Reply:
I have a use case where this is what I need. Why is there a general need to more? We can always add stuff, it is not like we are finalizing things. I think you are weirdly gatekeeping. Why should i return the number of pages? Its not relevant to what i am doing? 

Someone else later can just add it they need it.